views/layout-form-clear.twig

```diff
@@ -0,0 +1,27 @@
+
+{% extends "form-base.twig" %}
+{% import "forms.twig" as forms %}
+
+{% block formTitle %}
+    {% set layout = layout.layout %}
+    {% trans %}Clear Layout - {{ layout }}{% endtrans %}
+{% endblock %}
+
+{% block formButtons %}
+    {% trans "No" %}, XiboDialogClose()
+    {% trans "Yes" %}, $("#layoutClearForm").submit()
+{% endblock %}
+
+{% block formHtml %}
+    <div class="row">
+        <div class="col-md-12">
+            <form id="layoutClearForm" class="XiboForm form-horizontal" method="clear" action="{{ url_for("layout.clear", {id: layout.layoutId}) }}">
+                {% set message %}{% trans "Are you sure you want to clear this item?" %}{% endset %}
```

Comment:
I think we might want to specifically say Layout and maybe call out that it will be the Draft that is cleared?